Automattic/newspack-plugin
# feat(connections-fivetran): add TOS acceptance

### All Submissions:






### Changes proposed in this Pull Request:



Adds Newspack TOS acceptance checkbox to Fivetran connections. 

### How to test the changes in this Pull Request:

1. Set up Fivetran* Connections
2. Visit the Connections wizard, observe an unchecked Terms of Service acceptance checkbox
3. Observe the Fivetran cards' "Connect" links are disabled until TOS are accepted
4. Accept the TOS and observe the services can be connected now

\* provide a valid Fivetran proxy URL as an environment variable

### Other information:
